src/gen_smtp_application.erl

```diff
@@ -9,7 +9,7 @@ ensure_all_started(App) ->
 
 -spec start_ok(atom(), ok | {error, term()}) -> ok | {error, {term(), atom()}}.
 start_ok(_App, ok) -> ok;
-start_ok(_App, {error, {already_started, _App}}) -> ok;
+start_ok(App, {error, {already_started, App}}) -> ok;
 start_ok(App, {error, {not_started, Dep}}) ->
```

Comment:
not related to this PR, but in general, I think we don't need this module anymore. `application:ensure_all_started` was added to OTP R16, and gen_smtp does not support such old releases anymore

https://github.com/gen-smtp/gen_smtp/commit/beeeef71fa655076fe17f07ce243e0ad2a83b486

Reply:
You right. I think I did the same fix for `lager` one of these days.